test/component_model_tests.jl

```diff
@@ -0,0 +1,70 @@
+# this folder contains temporary tests for component code that has not been moved to any src code (location TBD)
+using Test
+using ClimaCore
+using ClimaCore: Fields
+using ClimaCoupler.Regridder: binary_mask
+
+include("TestHelper.jl")
+
+# sea ice
+include("../experiments/AMIP/modular/components/ocean/slab_seaice_init.jl")
+
+for FT in (Float32, Float64)
+    @testset "test sea-ice energy slab for FT=$FT" begin
+        function test_sea_ice_rhs(; F_rad = 0.0, T_base = 271.2, global_mask = 1.0)
+            space = TestHelper.create_space(FT)
+            params = IceSlabParameters(
+                FT(2),  # ice thickness [m]
+                FT(900.0), # density of sea ice [kg / m3]
+                FT(2100.0), # specific heat of sea ice [J / kg / K]
+                FT(T_base), # temperature of sea water at the ice base
+                FT(1e-3), # roughness length for momentum
+                FT(1e-5), # roughness length for tracers
+                FT(271.2), # freezing point of sea water
+                FT(2.0),# thermal condictivity of ice [W / m / K] (less in HM71)
+                FT(0.8), # sea ice albedo
```

Comment:
(like earlier comment) It might be more helpful to have the `IceSlabParameters` field names in comments here.